exercises/01-rest-api.md

```diff
@@ -43,30 +48,95 @@ should check the following:
 ### Step-by-step
 
 <details>
-  <summary>RESTinstance</summary>
+  <summary>Initialize you test suite</summary>
 
-#### RESTinstance
-
-**Initialize your test suite with `REST` library.**
+<br/>
 
 In this exercise we're not going to write very sophisticated Robot Framework, meaning
 that we're going to do very simple test cases without doing a resource file separately.
 In our `tests` folder, we have a file called `api.robot`. Let's open that up.
 
+> The file resources `../resources/bad_flask_app.robot`. That import is only used with `Browser`
+> library. If you're doing the exercises with `RESTinstance`, the import can be safely ignored. Running the
+> suite with an empty resource file will log a warning, but will not affect the outcome of the exercise.
+
+<details>
+  <summary>RESTinstance</summary>
+
 We're going to use the RESTinstance library, so we need to import `REST` into our `Settings`
-table. We'll test the REST API of the Bad Flask App. The server is running in `http://localhost:5000`,
-so let's initialize the library import with that URL.
+table. RESTinstance sends requests directly into the target server, so it
+requires a URL with the library import to initialize the library to do
+queries against that server. We'll test the REST API of the Bad Flask App.
+The server is running at `http://localhost:5000`, so let's initialize the library import with that URL.
 
 - Add a library import for `REST` in your `Settings` table.
 - Add `http://localhost:5000` as an argument for your library import.
 
+</details> <!-- RESTinstance -->
+
+<details>
+  <summary>Browser</summary>
+
+We're going to use the Browser library, so we need to import it into our `Settings` table in our resource file.
+We'll test the REST API of the Bad Flask App. In order to do that, we're going to need a new browser, because
+this library uses the browser instance to execute API calls.
+From the Browser library documentation we see that there's two possible keywords for this: `New Browser` and
+`New Page`. `New Browser` allows us to specify a browser and whether we want to use headless or headful
+along with a bunch of other configurations. `New Page` just opens a new tab in our browser (or calls
+`New Browser` with default parameters if no browser is open) to a URL we
+specify. Since we're just using REST API backend, we don't need to see a browser, so we can call `New Page`
```

Comment:
It sounds as if headless is default for new browser instance. Is this mentioned anywhere, just to be 100% clear?

Reply:
Headless is default yes. It is mentioned in Browser library documentation and in exercise 02 optional task of parameterising headful/headless browser windows.